README.md

```diff
@@ -160,7 +160,12 @@ From a design standpoint, the objective with `bfe` was to create the simplest
 to create/edit BIBFRAME data.  The current focus is to transform bfe into a production ready tool.
 
 All contributions are welcome.  If you do not code, surely you will discover an
-[issue] you can report.  
+[issue] you can report.
+
+#### User Authentication with AWS Cognito
+
+We currently use AWS Cognito to manage the authentication of users. Cognito uses a client id to manage the connection to 
+this application (see `awsClientID()` in `src/Config.js`). Whenever changes are made to the AWS configuration this client_id changes and so you must also submit a PR to change the client id in this application as well. In the production environment this will be handled via an environment variable, but for local functionality and testing, the hard-coded value must be updated, or you must set the environment variable `AWS_CLIENT_ID` to be the current client id.
```

Comment:
maybe no line break, or all line breaks?  (whatever is the custom in this README?)

Reply:
👌